.github/workflows/draft.yml

```
@@ -0,0 +1,36 @@
+# Based on https://github.com/release-drafter/release-drafter/blob/master/README.md
+name: Release Drafter
+on:
+  push:
+    # branches to consider in the event; optional, defaults to all
+    branches:
+      - master
```

```suggestion
      - main
```

But can you add a Debug commit to point it to `release-drafts` and show how it behaves on your fork?